gen3/lib/utils.sh

```diff
@@ -103,6 +103,67 @@ semver_ge() {
   ) 1>&2
 }
 
+# Takes 2 required and 1 optional arguments:
+#   $1 service name
+#   $2 version of service where tests apply >=
+#   $3 version of service where tests apply >=, in monthly release (2020.xx) format
+#
+# ex: isServiceVersionGreaterOrEqual "fence" "3.0.0"
+# or: isServiceVersionGreaterOrEqual "fence" "3.0.0" "2020.01"
+isServiceVersionGreaterOrEqual() {
```

Comment:
this logic was mostly copied and slightly modified from a similar feature in gen3-qa integration tests (for only running certain tests above a given version)